### DISCUSSION TITLE
Welcome to SlimHUD Discussions!

### DISCUSSION THREAD

**Discussion by AlexPerathoner** [AlexPerathoner/SlimHUD#25] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aremmell** [comment]
Off-Topic AF:

Love your work here; I plan to use it on all my macOS devices. I was looking closely at your screenshots and I noticed icons in the menu bar I don't recognize. What program(s) generate those (they  look like system health indicators, and maybe a Homebrew menu?)

**Reply by AlexPerathoner** [reply]
Hi!
here's a complete list of the apps I have in my menu bar, from left to right as seen in the screenshots:
- Discord
- Gif Brewery 3 (convert video to gif)
- Gestimer (quickly start timers by dragging down the icon - really cool)
- Little Snitch (Firewall)
- iStat Menus - network usage, disk usage, cpu usage, sensors, battery
- macos default volume output
- Bartender (click on its icon to hide/show even more menu bar icons, useful when you have so many)
- CornerCal (click on it to show a window with a month calendar)
- MonitorControl (use FN buttons to control brightness & volume of monitor)
- macos default Control Center
- macos default time / calendar (can't be hidden entirely, would rather completely replace it with CornerCal)

Bartender hides the following others:
- Bandwith+ (total network usage counter)
- BetterTouchTool
- OctoMouse (count mouse clicks & key strokes)
- Finicky (open different url domains with different browsers)
- Karabiner (override keyboard keys with others)
- Vinduet (self made Magnet clone for windows management. Has some more combinations like screen corners etc. There are some other open source which are definitely better)
- Rocket (inline emoji in any app/website by typing ":")
- MusicBar (self made utility app to pause music if volume is muted)

Hope this helps :)

**Reply by aremmell** [reply]
Thanks for that info. I will check those out!

**Comment by Simos805** [comment]
Hello! I installed SlimHUD on my 14" M3 Max MacBook Pro running the latest version of macOS Sonoma 14.4.1. When using the Grow animation, adjusting the display brightness makes the animation glitch and the bar appears to move from slightly the left to the right.
